evap/staff/views.py

```diff
@@ -815,9 +816,24 @@ def course_comments(request, semester_id, course_id):
             section_list = course_sections if contribution.is_general else contributor_sections
             section_list.append(CommentSection(questionnaire, contribution.contributor, contribution.label, contribution.responsible, text_results))
 
-    template_data = dict(semester=semester, course=course, course_sections=course_sections,
-            contributor_sections=contributor_sections, filter_comments=filter_comments)
-    return render(request, "staff_course_comments.html", template_data)
+    template_data = dict(semester=semester, course=course, view=view)
+
+    if view == 'quick':
+        visited = request.session.get('review-visited', set())
+        visited = set(visited)
```

Comment:
this shouldn't be necessary afaics

Reply:
Great eye! This remained as a in-dev-hack while migrating from lists to sets